airflow/dag_processing/bundles/git.py

```diff
@@ -163,12 +167,18 @@ def _clone_bare_repo_if_required(self) -> None:
             raise AirflowException(f"Connection {self.git_conn_id} doesn't have a host url")
         if not os.path.exists(self.bare_repo_path):
             self.log.info("Cloning bare repository to %s", self.bare_repo_path)
-            Repo.clone_from(
-                url=self.repo_url,
-                to_path=self.bare_repo_path,
-                bare=True,
-                env=self.hook.env,
-            )
+            try:
+                Repo.clone_from(
+                    url=self.repo_url,
+                    to_path=self.bare_repo_path,
+                    bare=True,
+                    env=self.hook.env,
+                )
+            except GitCommandError as e:
+                # log the error to appear in dag-processor stdout and raise exception to appear
+                # in the dag-processor logs
```

Comment:
fyi, these will be combined here soon - since it's no longer a standalone component, it'll all end up in stdout/stderr. Let's leave the log part out in prep.